ibridges/irodsconnector/data_operations.py

```diff
@@ -256,7 +256,7 @@ def _download_collection(session: Session, irods_path: Union[str, IrodsPath], lo
     for source, dest in source_to_dest:
         # ensure local folder exists
         if not dest.parent.is_dir():
-            os.makedirs(dest)
+            os.makedirs(dest.parent)
```

Comment:
Good catch!

Reply:
Testing on Windows sometimes helps to stabilise the code ;)